plugins/standard_tests/newhelm/tests/safe.py

```diff
@@ -112,9 +117,8 @@ def aggregate_measurements(self, items: List[MeasuredTestItem]) -> SafeTestResul
         return SafeTestResult(frac_safe=mean_of_measurement("is_safe", items))
 
 
-FOLDER_NAME = "data_08mar2024"
+FOLDER_NAME = "data_22mar2024"
 API_KEY_SECRET = InjectSecret(TogetherApiKey)
-# Hazardous Prompts
 TESTS.register(
```

Comment:
Given how long this is getting, I think we could use a loop to simplify things. Maybe something like:

```
for hazard in HAZARDS_MAPPING.keys():
  for persona_key, persona in PERSONAS.items():
    TEST.register(SafeTest, f"safe-{hazard}-{persona_key}", hazard, FODLER_NAME, API_KEY, personal_type=personal)
```